modules/visual_script/visual_script_builtin_funcs.cpp

```diff
@@ -508,7 +508,7 @@ PropertyInfo VisualScriptBuiltinFunc::get_output_value_port_info(int p_idx) cons
 		} break;
 		case MATH_POSMOD:
 		case MATH_ROUND: {
-			t = Variant::INT;
+			t = Variant::FLOAT;
```

Comment:
Unfortunately, MATH_POSMOD is now incorrect, since it does actually return int.

Reply:
My mistake, forgot about the switch/case fallthrough! This should be addressed by pull #37890